app/frontend/src/internationalization/trans/chefs/en/en.json

```diff
@@ -391,6 +391,7 @@
     "emailRequired": "Email is required"
   },
   "submissionsTable": {
+    "noMatchingRecordText": "No matching records found",
```

Comment:
I re-ordered the items in the en translation so that it's the same order as the other files. This makes it easier to compare files to check for inconsistencies.

I'm thinking, though, that we should do some sort of automation here, like alphabetizing the keys. That way it would always be easy to compare files for missing/extra/different keys.

Reply:
I think this should be done inside the main branch, but yes, I agree with you. Would make it much easier to search through the localization files.